unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/Voxels/VoxelEntityHit.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class VoxelEntityHit 
+{
+    public DecentralandEntityToEdit entityHitted;
+    public Vector3Int hitVector;
+
+    public VoxelEntityHit(DecentralandEntityToEdit entity, RaycastHit hit)
+    {
+        entityHitted = entity;
+        CalculateHitVector(hit);
+    }
+
+    void CalculateHitVector(RaycastHit hit)
+    {
+        Vector3 center = entityHitted.rootEntity.meshesInfo.mergedBounds.center;
+        Vector3 min = entityHitted.rootEntity.meshesInfo.mergedBounds.min;
+        Vector3 max = entityHitted.rootEntity.meshesInfo.mergedBounds.max;
+
+        if (Mathf.Abs(min.x -hit.point.x) < 0.001f)
```

Comment:
Let's move `0.001f` and all the occurences to a local const `HIT_VECTOR_SENSITIVITY` or something similar